drivers/media/platform/bcm2835/bcm2835-unicam.c

```diff
@@ -1815,12 +2058,14 @@ static int unicam_open(struct file *file)
 		goto unlock;
 	}
 
+	node->open++;
 	if (!v4l2_fh_is_singular_file(file))
 		goto unlock;
 
 	ret = v4l2_subdev_call(dev->sensor, core, s_power, 1);
 	if (ret < 0 && ret != -ENOIOCTLCMD) {
 		v4l2_fh_release(file);
+		node->open--;
```

Comment:
Minor nitpick, why not just increment node->open at the end alongside the ret = 0; ?

Reply:
Done.